core/sql/optimizer/CostMethod.h

```diff
@@ -2036,30 +2037,31 @@ class CostMethodHbaseInsert : public CostMethod
 //<pb>
 /**********************************************************************/
 /*                                                                    */
-/*                         CostMethodDP2UpdateOrDelete                */
+/*                         CostMethodHbaseUpdateOrDelete              */
 /*                                                                    */
 /**********************************************************************/
-class CostMethodDP2UpdateOrDelete : public CostMethod
+class CostMethodHbaseUpdateOrDelete : public CostMethod
 {
 public:
   // Constructor
-  CostMethodDP2UpdateOrDelete( const char* className )
+  CostMethodHbaseUpdateOrDelete( const char* className )
     : CostMethod( className ) 
   {};
 
 protected:
 
-  // Cost functions
+  // Old model cost function (obsolete; only here because base
+  // class requires an implementation)
   virtual Cost* computeOperatorCostInternal(RelExpr* op,
                                             const Context* myContext,
-                                            Lng32& countOfStreams) = 0;
+                                            Lng32& countOfStreams);
+
   // SCM Cost function
   virtual Cost* scmComputeOperatorCostInternal(RelExpr* op,
-					       const Context* myContext,
                                                const PlanWorkSpace* pws,
 					       Lng32& countOfStreams) = 0;
 
-  // Common functions to cost DP2 operations.
+  // Common functions to cost Hbase operations.
```

Comment:
Maybe remove the function declaration since it has not been implemented in Trafodion.